### PR TITLE
Remove outer paper in MaterialArrayLayout - closes #1520

### LINKED ISSUE #1520: Lot of paper components with nested arrays
I'm building a questions/answers form for our backend. This is what it looks like right now:

Screen Shot 2019-11-15 at 13 27 27

As you can see, all the nested paper components look a little distracting. I'd prefer it if only the actual array elements were embedded in a paper, like this:

Screen Shot 2019-11-15 at 13 29 50

What do you think? 🙂

Also, I didn't quite understand why the arrays are rendered so differently than in the examples on the website. I don't necessarily need an accordion. Is there an option I can set to disable the heavy nesting?

Thank you! 😊 

<details>
<summary>Configuration</summary>

```JS
const schema = {
    type: 'object',
    properties: {
        text: { type: 'string' },
        questions: {
            type: 'array',
            items: {
                type: 'object',
                properties: {
                    text: {
                        type: 'object',
                        properties: {
                            de: { type: 'string' },
                            fr: { type: 'string' },
                        }
                    },
                    answers: {
                        type: 'array',
                        items: {
                            type: 'object',
                            properties: {
                                text: {
                                    type: 'object',
                                    properties: {
                                        de: { type: 'string' },
                                        fr: { type: 'string' },
                                    }
                                },
                                score: { type: 'integer' }
                            },
                        },
                    },
                },
            },
        },
    },
};
const uischema = {
    type: 'VerticalLayout',
    elements: [
        {
            type: 'Control',
            scope: '#/properties/text',
        },
        {
            type: 'Control',
            scope: '#/properties/questions',
            options: {
                detail: {
                    type: 'VerticalLayout',
                    elements: [
                        {
                            type: 'HorizontalLayout',
                            elements: [
                                {
                                    type: 'Control',
                                    scope: '#/properties/text/properties/de'
                                },
                                {
                                    type: 'Control',
                                    scope: '#/properties/text/properties/fr'
                                },
                            ],
                        },
                        {
                            type: 'Control',
                            scope: '#/properties/answers',
                            options: {
                                detail: {
                                    type: 'VerticalLayout',
                                    elements: [
                                        {
                                            type: 'HorizontalLayout',
                                            elements: [
                                                {
                                                    type: 'Control',
                                                    scope: '#/properties/text/properties/de'
                                                },
                                                {
                                                    type: 'Control',
                                                    scope: '#/properties/text/properties/fr'
                                                },
                                            ],
                                        },
                                        {
                                            type: 'Control',
                                            scope: '#/properties/score'
                                        },
                                    ],
                                },
                            },
                        },
                    ],
                },
            },
        },
    ],
};
```

</details>

### PR DESCRIPTION
**Pull request by iluuu1994**

### CODE REVIEW

**Comment by sdirix** [comment]
Thank you very much for the contribution. Great that you took the topic into your own hands ;) Much appreciated! We'll take a closer look at it soon.

**Comment by iluuu1994** [comment]
I made a small adjustment to also remove the gutters from the array toolbar.

Before:

Screen Shot 2019-11-20 at 21 43 08

After:

Screen Shot 2019-11-20 at 21 41 48

**Comment by sdirix** [comment]
Works for me!

I discussed the change with the team. In most cases we like how it looks however in some cases (mainly when there is a lot of space) the title and add button look somewhat lost and it's hard to parse visually that they belong to the array.

We'll merge this anyway as we have the very same problem already with the table array renderer.

Do you have a good idea for visually connecting the table array toolbar to its children?

**Comment by iluuu1994** [comment]
My form is very small so it didn't occur to me. Do you have an example where this is the case? Often spacing helps (more `margin-top`, less `margin-bottom`).

**Comment by sdirix** [comment]
@iluuu1994 I created a follow up issue: https://github.com/eclipsesource/jsonforms/issues/1524

**Comment by iluuu1994** [comment]
Thank you @sdirix!
